src/main/java/seedu/mark/logic/commands/ImportCommand.java

```diff
@@ -0,0 +1,77 @@
+package seedu.mark.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Optional;
+import java.util.logging.Logger;
+
+import seedu.mark.commons.core.LogsCenter;
+import seedu.mark.commons.exceptions.DataConversionException;
+import seedu.mark.logic.commands.commandresult.CommandResult;
+import seedu.mark.logic.commands.exceptions.CommandException;
+import seedu.mark.model.Model;
+import seedu.mark.model.ReadOnlyMark;
+import seedu.mark.storage.JsonMarkStorage;
+import seedu.mark.storage.Storage;
+
+/**
+ * Imports Mark's {@code Bookmark} data from a specific file.
+ */
+public class ImportCommand extends Command {
+
+    public static final String COMMAND_WORD = "import";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Imports all of Mark's data from the specified file.\n"
+            + "Parameters: FILENAME\n"
+            + "Example: " + COMMAND_WORD + " savedBookmarks";
+
+    public static final String MESSAGE_IMPORT_SUCCESS = "Bookmarks successfully imported from %1$s";
+    public static final String MESSAGE_IMPORT_FAILURE = "There was a problem while trying to import bookmarks "
+            + "from this file";
+    public static final String MESSAGE_FILE_FORMAT_INCORRECT = "The format of data in the file %1$s is incorrect";
+    public static final String MESSAGE_FILE_NOT_FOUND = "The file %1$s does not exist";
+
+    private static final Logger logger = LogsCenter.getLogger(JsonMarkStorage.class);
```

Comment:
I havent figured out where to use logger yet. But if it is used in import command, then we should use it in export command as well for the sake of consistency.